common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -429,6 +428,11 @@ div.problem {
       }
     }
 
+    // Fix for formulaequationinput
```

Comment:
Nit: It would be great if you could flesh out this comment. What exactly does this fix? In 6 months no one will remember :(